test/integration/targets/wait_for/tasks/main.yml

```diff
@@ -114,12 +114,12 @@
     path: "{{remote_tmp_dir}}/utf16.txt"
     search_regex: completed
 
-- name: test non mmapable file
+- name: test non mmapable file, skip OSs w/o /sys
   wait_for:
     path: "/sys/class/net/lo/carrier"
     search_regex: "1"
     timeout: 30
-  when: ansible_os_family not in ['FreeBSD', 'Darwin']
+  when: ansible_os_family not in ('Darwin', 'FreeBSD')
```

Comment:
This change isn't needed.

Reply:
was not intended, was going back and forth, will undo